scss/civicrm/event/_detail.scss

```diff
@@ -0,0 +1,180 @@
+
+    #crm-submit-buttons {
+      padding-left: 5px;
+    }
+
+    .crm-event-is_show_location br {
+      display: none;
+    }
+
+    h3 {
+      @include section-title;
+      margin: 0 -20px 20px -15px;
+    }
+  }
+
+  .crm-event-manage-location-form-block > table {
+    margin-left: 15px;
+
+    .status {
+      padding-left: 0;
+    }
+  }
+
+  #priceSet .message {
+    margin-right: 15px;
+  }
+
+  #contributionType .description {
+    padding-right: 15px;
+  }
+
+  fieldset#discount,
+  fieldset#map-field {
+    padding-left: 20px;
+    padding-right: 20px;
+
+    > legend {
+      @include section-title;
+      display: block;
+      width: 100%;
+      margin-left: -20px;
+    }
+
+    table {
+      min-width: 100%;
+    }
+
+    tr {
+      background-color: transparent;
+      border-top: 1px solid $select2-inner-background;
+      display: block;
+      margin: 0 -20px;
+      padding: 0px 20px 0px 20px;
+    }
+
+    td {
+      border: 0;
+      display: inline-block;
+      padding: 5px 10px 9px;
+      width: 260px;
+    }
+
+    .dateplugin {
+      margin-top: 5px;
+    }
+
+    .columnheader {
+      display: block;
+      background-color: $gray-lighter;
+      border-bottom: 1px solid $crm-grayblue-dark;
+      border-left: 0;
+      border-radius: 0;
+      border-right: 0;
+      margin: 20px -20px 0;
+      padding: 6px 20px 4px 20px;
+
+      br {
+        display: none;
+      }
+
+      td {
+        color: $datepicker-gray-dark;
+        font-size: 13px;
+      }
+
+      label {
+        margin-left: 5px;
+      }
+    }
+  }
+
+  fieldset#discount {
+    .columnheader td:first-child,
+    .form-item td:first-child {
+      padding-left: 2px;
+      width: 20px;
+    }
+
+    .crm-form-submit {
+      margin: 10px 20px 0;
+    }
+  }
+
+  #isDiscount {
+    padding-left: 20px;
+    padding-right: 20px;
+
+    .crm-event-manage-fee-form-block-is_discount td {
+      width: auto;
+    }
+  }
+
+  .crm-collapsible {
+    &:not(.collapsed) .collapsible-title:before {
+      content: '\f107';
+    }
+
+    &.collapsed .collapsible-title:before {
+      content: '\f105';
+    }
+
+    .collapsible-title + div,
+    #confirm_screen_settings {
+      padding-bottom: 20px;
+      padding-top: 20px;
+    }
+
+    .collapsible-title + table {
+      margin: 20px 0;
+      width: 100%;
+    }
+
+    .collapsible-title {
+      @include accordion-header;
+      font-size: $font-size-base;
+      margin-bottom: 0;
+    }
+  }
+
+  #Registration .crm-submit-buttons {
+    padding: 8px 20px 20px;
+
+    #_qf_Registration_upload_done-bottom {
+      padding-left: 26px;
+    }
+
+    .fa-check-circle {
+      padding-top: 4px;
+    }
+  }
+
+  .form-layout td.label,
+  .form-layout-compressed td.label,
+  .crm-accordion-body td.label,
+  .crm-form-block td.label {
+    vertical-align: middle;
+
+    label {
+      margin: 0;
+    }
+  }
+}
+
+&-add #crm-container .crm-form-block.crm-form-block>.crm-submit-buttons {
```

Comment:
what is this `&` referring to in this case? This looks a very weird selector

Reply:
Check this file: https://github.com/rocksolid-psf/org.civicrm.shoreditch/blob/73d4f51fdae99c97ded704b0fea030fa4e10ca8e/scss/civicrm/event/_event.scss

```
.page-civicrm-event {
  &-add {}
}
```

Reply:
got it, thanks